src/skmultiflow/trees/nodes/active_learning_node_for_regression_multi_target.py

```diff
@@ -30,7 +30,7 @@ def learn_from_instance(self, X, y, weight, ht):
             Targets values.
         weight: float
             Instance weight.
-        ht: HoeffdingTree
+        ht: HoeffdingTreeClassifier
```

Comment:
The same as before. My fault here. I forgot to change the description when implementing the node classes.

Reply:
Updated in 38da85c44c8c524c58704ec9e8718c554e313b0f